source/sass/views/youtube-regrets.scss

```diff
@@ -0,0 +1,21 @@
+#view-youtube-regrets {
```

Comment:
We are going to have lots of style overrides for this page so I decided to keep all the overrides here than creating a separate SCSS file each Wagtail component (e.g., "youtube regret" component).